chemper/graphs/fragment_graph.py

```diff
@@ -271,7 +273,7 @@ def add_atom(self, new_atom, new_bond=None, bond_to_atom=None,
 
         new_atom_storage = self.AtomStorage(new_atom, smirks_index=new_smirks_index)
         self._graph.add_node(new_atom_storage)
-        if new_smirks_index is not None and new_smirks_index > 0:
+        if new_smirks_index is not None:
```

Comment:
You removed a check that only does this if the index is greater than 0; deliberate? Why? (Just confirming.)

Reply:
yes, it was deliberate. Originally I was thinking of `atom_by_smirks_index` for only atoms that are actually tagged with smirks indices used in the smirks pattern (positive values), however, I realized for the cluster graph that using unique smirks indices made a convenient way to book keep atoms so I wanted the dictionary to have all of them.